src/web/toga_web/app.py

```diff
@@ -13,33 +14,246 @@ def __init__(self, interface):
         self.interface = interface
         self.interface._impl = self
 
-        self.create()
-
     def create(self):
-        self.interface.startup()
 
         self.interface.icon.bind(self.interface.factory)
         # self.resource_path = os.path.dirname(os.path.dirname(NSBundle.mainBundle.bundlePath))
 
         formal_name = self.interface.formal_name
+
         self.interface.commands.add(
+            # ---- Help menu ----------------------------------
             toga.Command(
-                self.interface.about_command,
-                'About ' + formal_name,
-                group=toga.Group.APP
+                lambda _: self.interface.about(),
+                "About " + formal_name,
+                group=toga.Group.HELP,
+            ),
+            toga.Command(
+                None,
+                "Preferences",
+                group=toga.Group.HELP,
             ),
-            toga.Command(None, 'Preferences', group=toga.Group.APP),
         )
 
+        # self.menubar.innerHTML = f"""
+        #     <nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
+        #         <div class="container">
+        #             <a class="navbar-brand" href="#">
```

Comment:
Does this comment serve any purpose?

Reply:
No - it was a prior implementation that I forgot to clean up. Fixed.